encryption/encryption.cc

```diff
@@ -197,6 +212,15 @@ Status NewAESCTRCipherStream(EncryptionMethod method, const std::string& key,
     case EncryptionMethod::kAES256_CTR:
       cipher = EVP_aes_256_ctr();
       break;
+    case EncryptionMethod::kSM4_CTR:
+    // Openssl support SM4 after 1.1.1 release version.
+    #if OPENSSL_VERSION_NUMBER < 0x1010100fL
```

Comment:
No indentation for macros.
```suggestion
#if OPENSSL_VERSION_NUMBER < 0x1010100fL
```